Stripe/StripeError+Internal.h

```diff
@@ -0,0 +1,17 @@
+//
+//  NSError+StripeInternal.h
+//  Stripe
+//
+//  Created by Ben Guo on 5/22/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface NSError (StripeInternal)
```

Comment:
Should we just make `StripeError` a subclass of `NSError` instead of a category? I think I've seen that pattern before.